pyspark/graphar_pyspark/enums.py

```diff
@@ -0,0 +1,68 @@
+"""
+Copyright 2022-2023 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from enum import Enum
+
+from py4j.java_gateway import JavaObject
+
+from graphar_pyspark import GraphArSession
+
+
+class GarType(Enum):
+    """Main data type in gar enumeration."""
```

Comment:
A little confusion here: Why here we use the enum with string value but not like scala with int value? Is there any benefit?

Reply:
It makes conversion from and to scala little simpler because in scala there is a method, that return string-representation of Enum. Also, for me `GarType("INT64")` looks better than `GarType(0)`. But I can change it, no problem! Should I?

Reply:
I don't know is there any efficiency performance influence between enum(string) and enum(int)，if the efficiency is  not affected, it's ok for me to conversion from scala with string.

Reply:
As I know, in Python each enum member is a singleton. So there is no more than one instance of each member at any point of time. And any usage of enum will be just a reference to corresponding object. In this case there should not be any significant difference between using strings and ints as member types. I will check it additionally by making a direct performance test.

Reply:
![image](https://github.com/alibaba/GraphAr/assets/29755009/693d3bf9-be59-4408-abf8-b45806de4051)
Looks like there is no significant difference, only some random fluctuations (I ran it few times)

Reply:
nice work, thanks for the explanation and efficiency check